core/modules/script.py

```diff
@@ -45,18 +82,33 @@ class ScriptNodeSkel(BaseScriptAbstractSkel):
         defaultValue=False,
     )
 
-    plugin = BooleanBone(
-        descr="Is plugin?",
-        defaultValue=False
-    )
-
```

Comment:
What is the reason for removing this bone?
Please first talk to @ciansen as Scriptor belongs to this bone.